udn-density-pods.go

```diff
@@ -47,12 +47,23 @@ func NewUDNDensityL3Pods(wh *workloads.WorkloadHelper) *cobra.Command {
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			setMetrics(cmd, metricsProfiles)
-			rc = wh.Run(cmd.Name())
+			if l2 && l3 {
+				// unset l3 if both are enabled.
+				l3 = false
```

Comment:
You can use the cobra function cmd.MarkFlagsMutuallyExclusive() to avoid this check

Reply:
https://github.com/kube-burner/kube-burner-ocp/blob/762ae83e3c9fee8d8c91515bbfa96ccc07df41af/custom-workload.go#L90-L91

Reply:
I still need this condition -- because I want to default to l3 (to avoid having to change what we have in CI right now)... so if the user passes layer2, I will just unset layer3.